fftest/vars.go

```diff
@@ -99,6 +117,58 @@ func Compare(want, have *Vars) error {
 	return nil
 }
 
+func CompareTest(t *testing.T, want, have *Vars) error {
+	t.Helper()
+
```

Comment:
This new section of `CompareTest()`, adding the `WantParseErrorIs` section is a good refactoring.  👍

Reply:
Oops!!! I meant to refactor that totally!!

Reply:
Thanks for pointing it out.